presto-product-tests/src/main/java/io/prestosql/tests/sqlserver/SqlServerDataTypesTableDefinition.java

```diff
@@ -51,7 +51,7 @@ private SqlServerDataTypesTableDefinition() {}
             return ImmutableList.<List<Object>>of(
                     ImmutableList.of(Long.MIN_VALUE, Short.MIN_VALUE, Integer.MIN_VALUE, Byte.MIN_VALUE,
                             Double.MIN_VALUE, Float.valueOf("-3.40E+38"), "\0", "\0", "\0", "\0", "\0", "\0",
-                            Date.valueOf("0001-01-02"), Timestamp.valueOf("1753-01-01 00:00:00.000"),
+                            Date.valueOf("1953-01-02"), Timestamp.valueOf("1753-01-01 00:00:00.000"),
                             Timestamp.valueOf("0001-01-01 00:00:00.000"), Timestamp.valueOf("1900-01-01 00:00:00"),
```

Comment:
Change these as well? I'd be fine if we change all the years to be `2000`+

Typo "minuts" in commit message

Reply:
This is weird, i don't understand this. However, let's change this timestamp value when it becomes a problem.
The good side with timestamp is that, should a test fail, toString would reveal the problem.